<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowRider</class>
 <widget class="QMainWindow" name="MainWindowRider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_Main">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Introduction">
        <item>
         <widget class="QLabel" name="label_hello">
          <property name="text">
           <string>Hello user!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_welcome">
          <property name="text">
           <string>Welcome to Eorzea Express</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_MapDisplay">
        <item>
         <widget class="QScrollArea" name="scrollArea_map">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>384</width>
             <height>228</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_map">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_displayBox">
          <item>
           <widget class="QLabel" name="label_display">
            <property name="text">
             <string>Trip Display</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_display"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_SelectionBox">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_driverBox">
          <item>
           <widget class="QLabel" name="label_drivers">
            <property name="text">
             <string>Select Driver</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_drivers">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>211</width>
               <height>225</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QListView" name="list_drivers"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_destinationFrom">
          <item>
           <widget class="QLabel" name="label_destinationFrom">
            <property name="text">
             <string>From Destination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>211</width>
               <height>225</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QListView" name="list_destFrom"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_destinationTo">
            <property name="text">
             <string>To Destination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>211</width>
               <height>225</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QListView" name="list_destTo"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_ConfirmationBox">
          <item>
           <widget class="QLabel" name="label_confirmation">
            <property name="text">
             <string>Confirmation</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_confirmation">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="button_add">
               <property name="text">
                <string>Add Destination</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_complete">
               <property name="text">
                <string>Confrim Trip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_cancel">
               <property name="text">
                <string>Cancel Trip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollArea" name="scrollArea_status">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents_5">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>91</width>
                  <height>69</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_status">
                   <property name="font">
                    <font>
                     <italic>true</italic>
                    </font>
                   </property>
                   <property name="text">
                    <string>status</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRider_Main_Menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="act_history"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="act_zoomIn"/>
    <addaction name="act_zoomOut"/>
    <addaction name="separator"/>
    <addaction name="act_normalize"/>
    <addaction name="act_fitToWindow"/>
   </widget>
   <addaction name="menuRider_Main_Menu"/>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="act_zoomIn">
   <property name="text">
    <string>Zoom in (25%)</string>
   </property>
   <property name="toolTip">
    <string>Zooms in on map by 25%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="act_zoomOut">
   <property name="text">
    <string>Zoom out (25%)</string>
   </property>
   <property name="toolTip">
    <string>Zooms out on map by 25%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="act_normalize">
   <property name="text">
    <string>Default Map Size</string>
   </property>
   <property name="toolTip">
    <string>Adjusts zoom to default image size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+-</string>
   </property>
  </action>
  <action name="act_fitToWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
   <property name="toolTip">
    <string>Fits map to window size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
  </action>
  <action name="act_history">
   <property name="text">
    <string>Transaction History</string>
   </property>
   <property name="toolTip">
    <string>History of user's past transactions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
